ci/crossbeam-epoch.sh

```diff
@@ -20,4 +20,9 @@ if [[ "$TRAVIS_RUST_VERSION" == "nightly" ]]; then
         --target x86_64-unknown-linux-gnu \
         --features sanitize,nightly \
         --example sanitize
+
+    # Check for no_std environment.
+    cd .. && cargo run --manifest-path ci/Cargo.toml --bin remove-dev-dependencies */Cargo.toml && cd -
```

Comment:
If there is a package that requires std in development dependency, it may not be checked properly in no_std environment. See https://github.com/rust-lang-nursery/futures-rs/issues/1396#issue-395720940 for more.

Reply:
This may not be necessary with the current crossbeam, so I will try to remove it once Travis green.

Reply:
Well, this was not necessary because the current crossbeam development dependency was the only rand. However, in the future, it may change.

@stjepang Do you think we should have this from now?